source/common/upstream/health_checker_base_impl.cc

```diff
@@ -104,14 +105,19 @@ std::chrono::milliseconds HealthCheckerImplBase::interval(HealthState state,
   } else {
     base_time_ms = no_traffic_interval_.count();
   }
+  return HealthCheckerImplBase::intervalWithJitter(base_time_ms, interval_jitter_);
```

Comment:
Do you need to qualify this with `HealthCheckerImplBase::`?

Reply:
nope